src/benchq/data_structures/hardware_architecture_models.py

```diff
@@ -116,14 +116,8 @@ def get_hardware_resource_estimates(self, resource_info: ResourceInfo):
             num_communication_qubits_per_elu=communication_qubits,
             num_memory_qubits_per_elu=memory_qubits,
             num_computational_qubits_per_elu=computational_qubits,
-            num_optical_cross_connect_layers=self.num_optical_cross_connect_layers(
-                num_elus,
-                communication_qubits,
-                num_communication_ports_per_elu,
-            ),
-            num_ELUs_per_optical_cross_connect=self.num_ELUs_per_optical_cross_connect(
-                code_distance, communication_qubits
-            ),
```

Comment:
Oh geeze I never realized how bad this was. 🤦‍♀️  Can you instead get rid of the warning when these functions are called and have them return something like "not implemented"? I think that would be a lot less confusing.

Reply:
Since I dont want to interfere with the interface changes, Ill keep the interface as int, and just return -1 as "unreasonable" value. Anything different than int could cause issues in other places, or the interface would have to be changed

Reply:
Ok yeah that's fine for now. The real answer is that the interface needs to be changed but we'll cross that bridge when we get to it.